### PR TITLE
Entity Channel

### PR DESCRIPTION
**Pull request by indiest**
Changelog:
- Moved the object replication from the spatial channel to the entity channel;
- Updated the handover logic to sync with channeld's group-based-handover;
- Implemented AActor::NetUpdateFrequency;
- Implemented the Well-Known Object (AActor::bAlwaysRelevant);
- Updated the codegen for the entity channel data;
- Updated the read/write logic of the packet tag to sync with https://github.com/metaworking/channeld/pull/36;
- Increased the FSocket's send/receive buffer size to 1MB;
- Fixed the infinite bounce issue of the RPC redirection;

### CODE REVIEW

**Comment by indiest** [comment]
@Augkit please review the change in the replication codegen

@laughxing please review the change in the transport layer (mainly `UChanneldConnection`). Any suggestion in other changes is also welcomed!

File: Source/ChanneldUE/ChanneldConnection.cpp [hunk 1]
```diff
@@ -407,49 +460,82 @@ void UChanneldConnection::TickOutgoing()
...
+		
+		if (bDisableMultiMsgPayload)
+		{
+			SendDirect(Packet);
+			Packet.clear_messages();
+		}
 	}
 
-	SendDirect(Packet);
+	if (Packet.messages_size() > 0)
```
Comment (laughxing) [inline Source/ChanneldUE/ChanneldConnection.cpp:496]: It appears that we are only able to transfer Channeld::MaxPacketSize data during each tick, which means that the throughput is limited to Channeld::MaxPacketSize*tickRate. This limitation becomes unacceptable when the network is under heavy load.

Comment (indiest) [reply Source/ChanneldUE/ChanneldConnection.cpp:496]: How much network load are we talking about? Do you think we should increase the MaxPacketSize?

Comment (laughxing) [reply Source/ChanneldUE/ChanneldConnection.cpp:496]: If the condition at line 476 is met, we break from the loop, send out the packet, and return immediately, without considering any remaining packets in the OutgoingQueue. We can keep sending until the Queue is empty.

Comment (indiest) [reply Source/ChanneldUE/ChanneldConnection.cpp:493]: I see. It is unnecessary to defer the remaining messages to the next TickOutgoing(). Code updated.

File: Source/ThirdParty/update_channeld.sh
Comment (Augkit) [inline Source/ThirdParty/update_channeld.sh:1]: The version of Setup.bat also needs to be modified.

**Review by Augkit — APPROVED** [review]

**Pull request merged**